### ISSUE TITLE
Investigate AutoKeras

### ISSUE THREAD

**Issue by Industrial** [Industrial/test-next.js#14]
https://autokeras.com/

**Comment by hemangjoshi37a** [comment]
How to add label to an issue??

**Comment by Industrial** [comment]
https://prnt.sc/m2s4uk

**Comment by Industrial** [comment]
AutoKeras does not apply (yet) to time-series
